frontend/src/metabase/css/components/buttons.module.css

```diff
@@ -25,7 +25,8 @@
 }
 
 .Button:focus {
-  outline: 2px solid var(--mb-color-focus);
+  border: 2px solid var(--mb-base-color-white);
```

Comment:
@iethree for whitelabelling/embedding purposes, shouldn't this be using the semantic "bg-primary" or whatever the token is called (which resolves to white in the core app), so that if you have this button on a dark background you wouldn't see a white border?